### DISCUSSION TITLE
RPM Lower when using Enhanced Exercise Daily mode.

### DISCUSSION THREAD

**Discussion by moisedo** [jgyates/genmon#1040] [Questions] [answered]
Use the template below if you have an issue or want to report a bug. If you have a question or a feature request you can ignore the questions below. This is a template so feel free to remove any of this text. Try to include the info below when applicable.

***NOTE:*** If you are having issues with your serial connection, please read this page before posting:

https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting

If you are having other issues, please see the following page:

https://github.com/jgyates/genmon/wiki/3.5---General-Troubleshooting

You can search past issues by removing the "is:open" keywords from the search window and adding any keywords that are relevant to your issue.

If you need to send you logs and registers to the developer, if your email is setup and working properly you can click send your logs on the About page in the web interface.

### Expected Behavior

It is expected for the generator to run normally at the correct RPM and frequency during the Daily Enhanced Exercise, similar to its performance when turned on via the controller Manual exercise via the maintenance page in the portal.

### Actual Behavior

During the Daily Enhanced Exercise scheduled at 10:30 AM daily, the generator's RPM is lower than expected, running at less than 2500 RPMs, whereas it should be around 3600 RPMs to match the normal operational standards. There are no error codes displayed when this issue occurs.

### Steps to Reproduce (including precondition)

1. Schedule the Daily Enhanced Exercise for 10:30 AM.
2. Observe the generator's RPM once it starts.
3. Note that the RPM is below the expected ~3600 RPMs, specifically under 2500 RPMs.
4. To fix the issue, i go to the Maintenance tab, stop the generator manually, then wait 10 secs, then Start it and it comes to the correct RPM.
5. Compare this performance with starting the generator via the controller or the portal, where it reaches the correct RPM threshold.

### Screenshot or Pictures relating to the problem (if possible)

N/A

### Your Environment

Generator Model: Generac 8KW/410 GUARD+10C T/SW - Model: 0058701
Genmon Logs: Submitted @ 10:38 AM EST Feb/6/2024.
Genmon Version: V1.19.01

**Comment by jgyates** [comment, marked as answer]
What you are describing is normal, assuming you have the Quite Mode exercise cycle selected.

There are three modes for the Enhanced Exercise Add on: Quite Mode, Normal and Transfer.

Quite Mode will run at a reduced RPM and output voltage. This is a feature of the controller. The built in exercie cycle has the option of doing this as well.
Normal will run at full speed (3600 RPM for Air cooled models)
Transfer will run at full speed but also engage the transfer switch.

Let me know if you have any questions.

**Reply by moisedo** [reply]
Thank you very much for this insight. It makes sense. QQ before i close this issue:

1. Is there an advantage/disadvantage to run it at a lower RPM (Quiet Mode) rather than Normal?

Thanks again for the prompt response.

**Reply by jgyates** [reply]
you uses slightly less less fuel and it is quieter.

**Reply by jgyates** [reply]
FYI, here is some more discussion on the topic:  https://www.zillerstore.com/post/quiet-mode-vs-regular-mode-22kw-9948597
